### PR TITLE
Clean up the MFA modal

### LINKED ISSUE #19042: remove popup when Per-session MFA  enabled
HI
When Per-session MFA is Enabled, each terminal connection shows a popup message "VERIFY YOUR IDENTITY" and we should click on "verify" and then two-factor-prompt. this is a "nerve-racking" feature

image

### PR DESCRIPTION
**Pull request by zmb3**
Change the text on the MFA dialog to be less alarming. Additionally, focus the OK button by default, so that users can press enter to go straight to the MFA prompt without manually clicking a button.

Closes #19042

### CODE REVIEW

File: web/packages/teleport/src/components/AuthnDialog/AuthnDialog.tsx [hunk 1]
```diff
@@ -41,13 +41,12 @@ export default function AuthnDialog({
...
         <Text textAlign="center">
-          Re-authentication is required. Follow the prompts given by your
-          browser to complete authentication.
+          Re-enter your multi-factor authentication in the browser to continue.
         </Text>
       </DialogContent>
       <DialogFooter textAlign="center">
-        <ButtonPrimary onClick={onContinue} mr={3} width="130px">
-          {errorText ? 'Retry' : 'Verify'}
+        <ButtonPrimary onClick={onContinue} setRef={e => e && e.focus()} mr={3} width="130px">
```
Comment (gzdunek) [inline web/packages/teleport/src/components/AuthnDialog/AuthnDialog.tsx:48]: `autoFocus` should be used:
```suggestion 
        <ButtonPrimary onClick={onContinue} autoFocus mr={3} width="130px">
```

**Review by gzdunek — APPROVED** [review]

File: web/packages/teleport/src/components/AuthnDialog/AuthnDialog.tsx (hunk 1 above)
Comment (ravicious) [inline web/packages/teleport/src/components/AuthnDialog/AuthnDialog.tsx:48]: The focus style of the primary button differs from a non-focused button only by the background color, so I suspect that users won't immediately know that the button is already focused.

But I believe changing that is outside of the scope of this PR.

**Review by ravicious — APPROVED** [review]
Looks fine to me, we should use `autoFocus` though as Grzegorz suggested. I just did a simple check and it works as expected here.

File: web/packages/teleport/src/components/AuthnDialog/AuthnDialog.tsx (hunk 1 above)
Comment (zmb3) [reply web/packages/teleport/src/components/AuthnDialog/AuthnDialog.tsx:48]: Nice, thanks!

Comment (zmb3) [reply web/packages/teleport/src/components/AuthnDialog/AuthnDialog.tsx:48]: Yes, out of scope, but worth mentioning to @roraback for design system updates. I noticed this too.
